docs/metrics/metrics.md

```diff
@@ -1,39 +1,16 @@
-# Publication of the metrics
+# Publication of the metrics in Prometheus on OCP
 
 In order to publish the metrics several steps need to be done:
-- Create a service monitor
-```yaml
-apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
-metadata:
-  name: k4e-operator-servicemonitor
-  namespace: k4e-operator-system
-  labels:
-    control-plane: controller-manager
-spec:
-  endpoints:
-    - port: metrics
-      interval: 30s
-      path: /metrics
-  selector:
-    matchLabels:
-      control-plane: controller-manager
-```
-- Add port 8080 to k4e-operator-controller-manager service
-```yaml
-    - name: metrics
-      protocol: TCP
-      port: 8080
-      targetPort: 8080
-```
-- Enable user-defined for user projects
-```yaml
+
+- Enable monitoring on the cluster
+```bash
+kubectl apply -f - <<EOF
 apiVersion: v1
-data:
-  config.yaml: |
-    enableUserWorkload: true
 kind: ConfigMap
 metadata:
   name: cluster-monitoring-config
   namespace: openshift-monitoring
+data:
+  config.yaml: |
+  EOF
```

Comment:
Isn't `enableUserWorkload: true` needed there?

Reply:
It isn't needed based on the admin guide and as verified by testing it.

https://docs.openshift.com/container-platform/4.8/monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-the-monitoring-stack

Reply:
Thanks! Interesting... I was following https://docs.openshift.com/container-platform/4.8/monitoring/enabling-monitoring-for-user-defined-projects.html but it seems to be inaccurate.